diagnosticsModule/Public/Get-AdfsServerConfiguration.ps1

```diff
@@ -44,132 +116,125 @@ Function Get-AdfsServerConfiguration
         }
-        catch [Exception]
-        {
-            $configurationOutput | Add-Member NoteProperty -name "ADFSAttributeStore" -value "SCRIPTERROR: $_.Exception.Message" -Force;
+        catch [Exception] {
+            $configurationOutput | Add-Member NoteProperty -name "ADFSAttributeStore" -value $null -Force;
         }
 
         try
         {
             $adfsCertificateCollection = Get-AdfsCertificateList -RemovePrivateKey
             $configurationOutput | Add-Member NoteProperty -name "ADFSCertificate" -value $adfsCertificateCollection -Force;
         }
-        catch [Exception]
-        {
-            $configurationOutput | Add-Member NoteProperty -name "ADFSCertificate" -value "SCRIPTERROR: $_.Exception.Message" -Force;
+        catch [Exception] {
+            $configurationOutput | Add-Member NoteProperty -name "ADFSCertificate" -value $null -Force;
         }
 
         try
         {
             $adfsClaimDescription = Get-AdfsClaimDescription -ErrorVariable adfsClaimDescription;
             $configurationOutput | Add-Member NoteProperty -name "ADFSClaimDescription" -value $adfsClaimDescription -Force;
         }
-        catch [Exception]
-        {
-            $configurationOutput | Add-Member NoteProperty -name "ADFSClaimDescription" -value "SCRIPTERROR: $_.Exception.Message" -Force;
+        catch [Exception] {
+            $configurationOutput | Add-Member NoteProperty -name "ADFSClaimDescription" -value $null -Force;
         }
         try
         {
             $adfsEndpoint = Get-AdfsEndpoint -ErrorVariable adfsEndpoint;
             $configurationOutput | Add-Member NoteProperty -name "ADFSEndpoint" -value $adfsEndpoint -Force;
         }
-        catch [Exception]
-        {
-            $configurationOutput | Add-Member NoteProperty -name "ADFSEndpoint" -value "SCRIPTERROR: $_.Exception.Message" -Force;
+        catch [Exception] {
+            $configurationOutput | Add-Member NoteProperty -name "ADFSEndpoint" -value $null -Force;
         }
         try
         {
             $adfsProperties = Retrieve-AdfsProperties
             $configurationOutput | Add-Member NoteProperty -name "ADFSProperties" -value $adfsProperties -Force;
         }
-        catch [Exception]
-        {
-            $configurationOutput | Add-Member NoteProperty -name "ADFSProperties" -value "SCRIPTERROR: $_.Exception.Message" -Force;
+        catch [Exception] {
+            $configurationOutput | Add-Member NoteProperty -name "ADFSProperties" -value $null -Force;
         }
-
+        
         try
         {
-            $adfsRelyingPartyTrustCount = 0
-            $adfsRelyingPartyTrustCount = (Get-AdfsRelyingPartyTrust).Count;
-
+            # Comment out to prevent heavy usage of SQL for customers with large RP sets for every run (1hr interval)
+            # $adfsRelyingPartyTrustCount = (Get-AdfsRelyingPartyTrust).Count;
```

Comment:
I am adding a check for Connect Health executing this test.  If Connect Health executes it we will skip getting the count but otherwise we execute it.